src/Symfony/Component/Image/Tests/Draw/AbstractDrawerTest.php

```diff
@@ -0,0 +1,253 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Image\Tests\Draw;
+
+use Symfony\Component\Image\Fixtures\Loader;
+use Symfony\Component\Image\Image\Box;
+use Symfony\Component\Image\Image\Font;
+use Symfony\Component\Image\Image\Palette\RGB;
+use Symfony\Component\Image\Image\Point;
+use Symfony\Component\Image\Image\Point\Center;
+use Symfony\Component\Image\Image\LoaderInterface;
+use Symfony\Component\Image\Tests\TestCase;
+
+abstract class AbstractDrawerTest extends TestCase
+{
+    public function testDrawASmileyFace()
+    {
+        $loader = $this->getLoader();
+
+        $canvas = $loader->create(new Box(400, 300), $this->getColor('000'));
+
+        $canvas->draw()
+            ->chord(new Point(200, 200), new Box(200, 150), 0, 180, $this->getColor('fff'), false)
+            ->ellipse(new Point(125, 100), new Box(50, 50), $this->getColor('fff'))
+            ->ellipse(new Point(275, 100), new Box(50, 50), $this->getColor('fff'), true);
+
+        $canvas->save(__DIR__.'/../results/smiley.png');
```

Comment:
you should either use a path outside the repo (in the system temp dir) or mark this folder as ignored.